README.md

```diff
@@ -1154,7 +1154,14 @@ machine twitch login my_twitch_account_name password my_twitch_password
 ```
 To activate authentication with the `.netrc` file you should pass `--netrc` to yt-dlp or place it in the [configuration file](#configuration).
 
-The default location of the .netrc file is `$HOME` (`~`). On Windows, if `$HOME` is not present, `$USERPROFILE` (generally `C:\Users\<user name>`) or `${HOMEDRIVE}${HOMEPATH}` is used
+The default location of the .netrc file is `${HOME}` (`~`).
+
+### Notes about environment variables
+* Environment variables are normally specified as `${VARIABLE}`/`${VARIABLE}` on UNIX and `%VARIABLE%` on Windows; but is always shown as `${VARIABLE}` in this documentation
```

Comment:
What is `${VARIABLE}`/`${VARIABLE}`? Is it a typo?

Reply:
It should be
```
`${VARIABLE}`/`$VARIABLE`
```

Reply:
Sorry, I didn't see that you fixed that